### PR TITLE
SAK-30370 present the ckeditor basic toolbar config to a user on a small device

### PR DESCRIPTION
**Pull request by ottenhoff**
I made one opinionated change here: I remove the ability for the tool "config" variable to override height and width. I think it's bad (legacy) behavior to allow the tool to define hard-coded values. The tool should define a container and the ckeditor will fill that container. Hard-coded values present serious usability issues for small-screen devices.

Nice and tiny editor for your tiny-screen device:

image

CC: @botimer and @jonespm

### CODE REVIEW

**Comment by botimer** [comment]
I think this is a definite step in the right direction. Especially since we are now dealing with many more screen sizes, more automatic behavior seems right.

In some ways, I would still like to see some notion of requesting a "basic" or "full" config for a given tool, but it may be that the screen size is the more appropriate determinant in general.

File: reference/library/src/webapp/editor/ckeditor.launch.js [hunk 1]
```diff
@@ -155,87 +154,46 @@ sakai.editor.editors.ckeditor.launch = function(targetId, config, w, h) {
...
-    //      "logical" configuration options is yet to be determined.
-    if (config) {
-        if (config.width) {
-            ckconfig.width = config.width;
-        } else if (w) {
-            ckconfig.width = w;
+    //To add extra plugins outside the plugins directory, add them here! (And in the variable)
+    (function() {
+        // SAK-30370 present a nice and simple editor without plugins to the user on a tiny screen.
+        if ($(window).width() < 768) {
```
Comment (juanjmerono) [inline reference/library/src/webapp/editor/ckeditor.launch.js]: Should this number (768) match with the morpheus phone media query break point value (800)? 
```
$break-small-end: 	800px;
$break-medium-end: 	1030px;

$phone  :       "only screen and (max-width : #{ $break-small-end })";
$tablet :       "only screen and (max-width : #{ $break-medium-end })";
$desktop:       "only screen and (min-width : #{ $break-medium-end + 1 })";
```

**Comment by ottenhoff** [comment]
@juanjmerono thank you. I changed the break point to 800px

@botimer I don't think we should allow tools to make the decision. Once we make every tool responsive, we should have a consistent CKeditor that appears the same way across all tools.

**Comment by botimer** [comment]
I'm persuaded by this approach, given that screen sizes are completely unpredictable now, and the difficulties presented by inconsistency over the years.

**Pull request merged**